src/components/step-flow/step-flow.mdx

```diff
@@ -46,6 +46,29 @@ An example of how this can be achieved is provided in the **Example implementati
 
 <Canvas of={StepFlowStories.DefaultStory} />
 
+### Step Flow Title node
+
+To allow a compositional approach, the `title` prop accepts a node. This can be used to compose custom content to be passed as the title of the `StepFlow` component.
+However, it is important to note that the `StepFlowTitle` sub component must be a descendant of the passed node, to ensure the component is accessible to screen reader users.
```

Comment:
**nitpick(non-blocking):** I _think_ subcomponent is all one word. If it's not then I'm demonstrating my ignorance to the world...

```suggestion
However, it is important to note that the `StepFlowTitle` subcomponent must be a descendant of the passed node, to ensure the component is accessible to screen reader users.
```